src/plugins/condition/components/Condition.vue

```diff
@@ -285,10 +280,13 @@ export default {
             return this.currentCriteria && identifier;
         },
         persist() {
-            this.openmct.objects.mutate(this.domainObject, 'configuration', this.domainObject.configuration);
+            this.$emit('updateCondition', {
+                condition: this.domainObject,
+                index: this.conditionIndex
+            });
         },
-        initCap: function (string) {
-            return string.charAt(0).toUpperCase() + string.slice(1)
+        initCap: function (value) {
+            return value.charAt(0).toUpperCase() + value.slice(1)
```

Comment:
I think `string` is actually more correct than `value` here. `str` if `string` is jarring for dev eyes, lol.

Reply:
Yes, string or str seem most appropriate.

Reply:
OK but I refuse to use a keyword here. :)

Reply:
Technically `string` is not reserved but I agree it is not a great variable name. Especially since we are discussing it.